webapp/components/FilterMenu/FilterMenu.vue

```diff
@@ -1,32 +1,23 @@
 <template>
-  <ds-card class="filter-menu-card">
-    <ds-flex class="filter-menu-content">
-      <ds-flex-item>
-        <ds-heading size="h3">{{ $t('filter-menu.hashtag-search', { hashtag }) }}</ds-heading>
-      </ds-flex-item>
-      <ds-flex-item>
-        <div class="filter-menu-buttons">
-          <base-button
-            name="clear-search-button"
-            icon="close"
-            circle
-            @click="clearSearch"
-            v-tooltip="{
-              content: this.$t('filter-menu.clearSearch'),
-              placement: 'left',
-              delay: { show: 500 },
-            }"
-          />
-        </div>
-      </ds-flex-item>
-    </ds-flex>
-  </ds-card>
+  <base-card class="filter-menu">
+    <h2>{{ $t('filter-menu.hashtag-search', { hashtag }) }}</h2>
+    <base-button
+      name="clear-search-button"
```

Comment:
```suggestion
      
```
This is no longer used, oder?
I think it was added for testing, but since you target `.base-card`, it should be removed.